v2/array/array.go

```diff
@@ -56,3 +56,13 @@ func DifferenceString(arrs ...[]string) (result []string) {
 
 	return
 }
+
+func StringArrayContains(arr []string, stringToCheck string) bool {
```

Comment:
`ContainsString(...)` would be a better name, due to its inside of the package array

Reply:
Yes, thanks